db/patch68.sql

```diff
@@ -3,4 +3,4 @@
 -- This resolves https://joindin.jira.com/browse/JOINDIN-725
 ALTER DATABASE joindin CHARACTER SET utf8 COLLATE utf8_unicode_ci;
 
-INSERT INTO patch_history SET patch_number = 68;
\ No newline at end of file
+INSERT INTO patch_history SET patch_number = 69;
```

Comment:
That file has no meaning at all though. The code changes I agree, but this needs to be run on the DB.When its been run a simple rename (no PR needed) and insering the line to keep the dev DB up to date would be fine. The current method is set up for 1 PR at a time (which is less than ideal for growth)